tests/nightly/test_np_large_array.py

```diff
@@ -76,3 +77,11 @@ def test_softmax():
         true_output = np.full((SMALL_Y, LARGE_X), (1 / input_data.shape[axis]))
         output = npx.softmax(input_data, axis=axis)
         assert_almost_equal(output.asnumpy(), true_output, rtol=1e-5, atol=1e-5)
+
+@pytest.mark.skip(reason="CI hasn't switch to ILP64 OpenBLAS yet")
```

Comment:
Should we move this to nightly test folder?
I was also thinking if we should have a decorator that we can use to mark tests as nightly tests. So we can keep operator tests in same files for maintainability but also not run as UTs for PR checks.

@access2rohit wdyt?

Reply:
I think this is in the nightly folder. Currently we are not using ilp 64 blas on any machine, so this test will fail on nightly as well. Hence I added the skip. Sorry about the confusion.

Reply:
tests for same operator in same file will increase file size. I like the decorator idea but any specific reason to keep both tests in same file ?

@Zha0q1 I will review it in a while